index.js

```diff
@@ -25,7 +26,18 @@ exports.log = function(options, tags, logStatement) {
     if (key === 'msg' || key === 'message') {
       return;
     }
-    const val = typeof obj[key] === 'string' ? `"${obj[key]}"` : JSON.stringify(obj[key]);
```

Comment:
instead of switch, just replace JSON.stringify with safeJson

Reply:
and you can do typeof === object and then safeJson otherwise, just return obj[key]